package.json

```diff
@@ -36,25 +44,27 @@
     "fuse.js": "^3.1.0",
     "immutable-props": "^1.1.0",
     "isdev": "^1.0.1",
-    "knex": "^0.13.0",
+    "knex": "^0.14.0",
     "linklocal": "^2.8.1",
     "lodash": "^4.17.4",
     "prop-types": "^15.5.10",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
     "source-map-support": "^0.5.0",
-    "sqlite3": "^3.1.13",
+    "sqlite3": "https://github.com/mapbox/node-sqlite3/tarball/master",
```

Comment:
why are we doing this? have you verified this works on another computer by deleting their node_modules and reinstalling? If you want I can try it

Reply:
This is pulling off nightly bad idea. We should be using node-pre-gyp leave it as is.